### DISCUSSION TITLE
Unexpected Positioning of 'resourceTimeGridDay' within '.ec-button-group' Div

### DISCUSSION THREAD

**Discussion by Artlyve** [vkurko/calendar#338] [Q&A]
Hello,

When I initialize my formats 'timeGridWeek', 'listWeek', and 'resourceTimeGridDay', sometimes 'resourceTimeGridDay' appears inside or outside the .ec-button-group div. This is a problem for me because 'resourceTimeGridDay' changes position unexpectedly. Do you have any idea what might be causing this?
![image](https://github.com/user-attachments/assets/32201456-830f-4452-9f77-a57a83dec897)
![Out](https://github.com/user-attachments/assets/8e213523-2eff-4bed-b140-a241a01ffa4e)
![In](https://github.com/user-attachments/assets/cd271bdd-6f8f-4b52-b7e7-b9502532c48e)
![image](https://github.com/user-attachments/assets/d7fa2dda-0e41-4239-bcee-f8919253481a)

**Comment by vkurko** [comment]
Please see [headerToolbar](https://github.com/vkurko/calendar?tab=readme-ov-file#headertoolbar) description. Values separated by a comma will be displayed inside `.ec-button-group`. Values separated by a space will be displayed outside.
